modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -1,20 +1,26 @@
-= Creating an API Instance by Importing From Exchange
+= Add an API Instance From Exchange
```

Comment:
```suggestion
= Add an API Instance from Exchange
```

Reply:
Be sure to use the Chicago tab at https://capitalizemytitle.com/style/Chicago/.

Reply:
Done! Thanks!